gammapy/image/catalog.py

```diff
@@ -7,7 +7,7 @@
 from astropy.wcs import WCS
 from astropy.units import Quantity
 from astropy.table import Table
-from .maps import SkyMap
+from .maps import SkyImage
```

Comment:
Here I see you're still importing from `maps.py`. I think we said changing to `core.py` would be better, no?